src/components/home/FindUs.tsx

```diff
@@ -1,5 +1,28 @@
+"use client";
+
 const FindUs = () => {
-  return <div></div>;
+  return (
+    <div className="bg-vsa-yellow-100 flex flex-col items-center justify-center p-20">
+      <h1 className="text-vsa-brown font-vsa-main w-7/12 text-left text-5xl font-bold">
+        Find Us!
+      </h1>
```

Comment:
```suggestion
      <p className="text-vsa-brown font-vsa-main w-7/12 text-left text-5xl font-bold">
        Find Us!
      </p>
```
Make sure to not use h1 html tags since we are specifying a text size anyways the h1 is not necessary.